ci_framework/roles/kuttl_tests/tasks/deploy_kuttl.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Use https://github.com/openstack-k8s-operators/install_yamls/blob/master/devsetup/roles/download_tools/tasks/main.yaml?
+# Do we need https://code.engineering.redhat.com/gerrit/gitweb?p=openstack/tripleo-ci-internal-config.git;a=blob;f=playbooks/crc/openstack.yaml?
+
+- name: Download kubectl-kuttl
+  ansible.builtin.get_url:
+    url: "{{ cifmw_kuttl_tests_kubectl_kuttl_url }}"
+    dest: "{{ cifmw_kuttl_tests_kubectl_kuttl_location }}"
+    mode: '0755'
+    timeout: 30
```

Comment:
Since make [download_tools](https://github.com/openstack-k8s-operators/install_yamls/blob/69c6a29b1a7fb9e310cc645d9c17deadacbc87f9/devsetup/roles/download_tools/tasks/main.yaml#L68) already install kubectl-kuttl. Let's use that one.
```
- name: Download Install_yamls deps
  ansible.builtin.include_role:
       name: 'install_yamls_makes'
       tasks_from: 'make_download_tools'

Reply:
right - and since we're "in the project" we should know about the role location